src/components/viewers/XLSViewer.vue

```diff
@@ -1,116 +1,103 @@
 <template>
   <div class="text-viewer">
     <div class="wrapper">
-      <table v-if="parsed" style="width: 100%;">
+      <table v-if="parsed" style="width: 100%">
         <thead>
-        <tr>
-<!--          <th v-for="(header, key) in content.meta.fields"-->
-<!--              v-bind:key="'header-'+key">-->
-<!--            {{ header }}-->
-<!--          </th>-->
-        </tr>
+          <tr>
+            <th v-for="column in uniqueColumns" :key="column">{{ column }}</th>
```

Comment:
can we add a default value here for `column`, in case the value is not truthy. This is the current experience if column name is an empty string.
<img width="630" alt="Screenshot 2024-06-17 at 12 11 21 PM" src="https://github.com/Pennsieve/pennsieve-app/assets/160671504/e8fe3d67-4114-41bb-aee5-091eaf45ed26">

Reply:
updated - thanks